jsonenums.go

```diff
@@ -75,6 +75,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"./utils"
```

Comment:
@bfallik What would you recommend as an alternative? Importing it from clyphub does not work unless I use a stream of a few messy commits.